test/utils/createDOM.js

```diff
@@ -23,6 +23,7 @@ const blacklist = ['sessionStorage', 'localStorage'];
 function createDOM() {
   const dom = new JSDOM('', {
     pretendToBeVisual: true,
+    url: 'http://localhost',
```

Comment:
To prove that it has no effect. I don't know if we need this URL parameter to be honest, we could remove it.